sdk/vaa/payloads.go

```diff
@@ -402,6 +427,27 @@ func (r BodyWormholeRelayerSetDefaultDeliveryProvider) Serialize() []byte {
 	return serializeBridgeGovernanceVaa(WormholeRelayerModuleStr, WormholeRelayerSetDefaultDeliveryProvider, r.ChainID, payload.Bytes())
 }
 
+func (r BodyGeneralPurposeGovernanceEvm) Serialize() []byte {
```

Comment:
posting relevant comment for reviewers (including myself) https://github.com/wormhole-foundation/example-native-token-transfers/blob/16c10e51ab9b90f051bbd5f5aa69de1fcd94aa77/evm/src/wormhole/Governance.sol#L64-L67